estate_account/models/estate_property.py

```diff
@@ -0,0 +1,29 @@
+from odoo import Command, models
+
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    def action_set_sold(self):
+        move = self.env['account.move'].sudo().with_context(default_move_type='out_invoice').create(
```

Comment:
```suggestion
        move = self.env['account.move'].create(
```
you can specify the move_type directly in the fields below. For the sudo, you should be very careful when using it, and I don't think you need it here unless I'm mistaken